crates/flowctl/src/raw/suggest_schema.rs

```diff
@@ -149,7 +149,11 @@ pub async fn do_suggest_schema(
         };
 
     // The original collection schema to be used as the starting point of schema-inference
-    let schema_model = collection_def.schema.unwrap();
+    let schema_model = collection_def
+        .schema
+        .as_ref()
+        .or(collection_def.read_schema.as_ref())
```

Comment:
If we are looking at a materializatio ntask, this should be a `write_schema` though, right?